src/consensus/raft/raft.h

```diff
@@ -153,7 +153,7 @@ namespace raft
       ledger(std::move(ledger_)),
       channels(channels_),
 
-      distrib(-(int)election_timeout_.count(), 0),
+      distrib(0, (int)election_timeout_.count() / 2),
```

Comment:
does this make a difference anywhere or is it just easier to reason with like this?

Reply:
This halves the election times for our existing tests (when the arg is `12s`, an election happens between 6s and 12s rather than 12s and 24s). I expected to need to go and double all of these values, but the tests seem happy so I haven't.